src/components/card/PermissionFiltersCard.vue

```diff
@@ -0,0 +1,202 @@
+          <q-item>
+            <q-item-section class="text-grey">
+              {{ $t('PermissionFiltersCard.text.noResults') }}
+            </q-item-section>
+          </q-item>
+        </template>
+        <template #prepend>
+          <q-icon :name="$t('PermissionFiltersCard.icon.libraryId')" />
+        </template>
+      </q-select>
+    </q-card-section>
+  </q-card>
+</template>
+
+<script setup>
+import { ref, computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import * as LibraryService from 'src/services/LibraryService';
+
+defineEmits(['update:entity', 'update:action', 'update:library-id']);
+defineProps({
+  entity: {
+    type: String,
+    default: '',
+  },
+  action: {
+    type: String,
+    default: '',
+  },
+  libraryId: {
+    type: String,
+    default: '',
+  },
+});
+
+const { t } = useI18n();
+const inputDebounceTime = ref(process.env.INPUT_DEBOUNCE_TIME);
+const libraryName = ref('');
+const libraries = ref([]);
+const libraryOptions = ref([]);
+const actionOptions = computed(() => [
+  {
+    label: t('Permissions.text.CREATE'),
+    value: 'CREATE',
+  },
+  {
+    label: t('Permissions.text.DELETE'),
+    value: 'DELETE',
+  },
+  {
+    label: t('Permissions.text.UPDATE'),
+    value: 'UPDATE',
+  },
+  {
+    label: t('Permissions.text.ACCESS'),
+    value: 'ACCESS',
+  },
+]);
+const entityOptions = computed(() => [
+  {
+    label: t('Permissions.text.ADMIN'),
+    value: 'ADMIN',
+  },
+  {
+    label: t('Permissions.text.PROJECT'),
+    value: 'PROJECT',
+  },
+  {
+    label: t('Permissions.text.COMPONENT'),
+    value: 'COMPONENT',
+  },
+  {
+    label: t('Permissions.text.DIAGRAM'),
+    value: 'DIAGRAM',
+  },
+  {
+    label: t('Permissions.text.LIBRARY'),
+    value: 'LIBRARY',
+  },
+  {
+    label: t('Permissions.text.PROJECT_TEMPLATE'),
+    value: 'PROJECT_TEMPLATE',
+  },
+  {
+    label: t('Permissions.text.COMPONENT_TEMPLATE'),
+    value: 'COMPONENT_TEMPLATE',
+  },
+  {
+    label: t('Permissions.text.DIAGRAM_TEMPLATE'),
+    value: 'DIAGRAM_TEMPLATE',
+  },
+]);
+
+/**
+ * Filters the library options based on a given label.
+ * @param {object} libraryLabel - Library label.
+ * @param {Function} update - Update function used to update the state
+ * with the filtered library options. It should accept an async callback function as its argument.
+ */
+async function filterLibrary(libraryLabel, update) {
+  update(async () => {
+    const lowerCaseValue = libraryLabel.toLocaleLowerCase();
+
+    if (libraryName.value !== '') {
+      await LibraryService.find({
+        name: `lk_*${libraryName.value}*`,
+      }).then((data) => {
+        libraries.value = data.content.map((library) => ({
+          label: library.name,
+          value: library.id,
+        }));
+      });
+    } else {
+      libraries.value = [];
+    }
+
+    libraryOptions.value = libraries.value
+      .filter((library) => library.label.toLocaleLowerCase().indexOf(lowerCaseValue) > -1);
```

Comment:
remove the filter, because its already done by the api.

Reply:
You can removde all this lines and
instead of 
```
        libraries.value = data.content.map((library) => ({
          label: library.name,
          value: library.id,
        }));
```

you can :

```
        libraryOptions.value = data.content.map((library) => ({
          label: library.name,
          value: library.id,
        }));
```

Reply:
done